src/main/kotlin/day05/Collection.kt

```
@@ -0,0 +1,26 @@
+        println(it)
+    }
+
+    val wizard = TestWizard()
+    wizard.attack()
+    wizard.fireball { damage ->
+        println(damage)
+    }
+}
+
+// 확장 함수 예시
+class TestWizard() {
```

detekt (1.23.7)

[warning] An empty default constructor can be removed.

(detekt.empty-blocks.EmptyDefaultConstructor)